### ISSUE TITLE
[BUG] Font color is black instead of kirby-black

### ISSUE THREAD

**Issue by bommariusser** [kirbydesign/designsystem#2033]
## Describe the bug
Page-title and other places in Kirby and also seen in DRB where font isn't kirby-black (#1c1c1c) but #000000

## Describe how to reproduce the bug

1. Go to https://cookbook.kirby.design/#/home/showcase/page
2. Inspect color on page-title

## Which Kirby version was used?
5.0.20

## What was the expected behavior?
font color should be kirby-black

## Add any screenshots
![Skærmbillede 2022-02-02 kl  19 16 14 (1)](https://user-images.githubusercontent.com/49033002/152339469-130f784b-046d-4f98-8b97-8e5d331d2128.png)

![Skærmbillede 2022-02-02 kl  19 30 19 (1)](https://user-images.githubusercontent.com/49033002/152339466-c78895b4-da80-40c2-ba31-3a75e6431b37.png)

## Please complete the following information:
- OS: [e.g. Windows, MacOS, iOS]
- Browser [e.g. Chrome, Safari]
- Browser version [e.g. 22]

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/master/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Verified, looks like we are not correctly setting the `--ion-text-color` on `ion-content`. 

![image](https://user-images.githubusercontent.com/42470636/152349337-c0ca425c-9201-47e9-bbd0-a5dec8f32f3f.png)

**Comment by bommariusser** [comment]
Det bugfix that @jkaltoft did, can be found here: https://cookbook.kirby.design/branch/kby-2033-font-color-black-instead-of-kirby-black/#/home/intro (for the page-title color)

unfortunately the toolbar color also has the wrong color ;) this might be another bug in another issue but would be awesome to close both bugs... :)
